src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -172,9 +175,19 @@ public void showNumberOfAccount(int accSize) {
 
     public void showNewAddedAccount(Account acc) {
         printLine();
```

Comment:
Show new account added, functions should at least sound grammatically correct. No need to be cryptic for no reason